test/parallel/test-eventtarget-whatwg-customevent.js

```diff
@@ -0,0 +1,40 @@
+// Manually converted from https://github.com/web-platform-tests/wpt/blob/master/dom/events/CustomEvent.html
+// in order to define the `document` ourselves
+
+// Flags: --expose-internals
+
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
+const common = require('../common');
```

Comment:
If this is going to be unused then this should just be 
```suggestion
require('../common');
```